src/main/webapp/app/course/manage/overview/course-management-overview-statistics.component.ts

```diff
@@ -21,90 +17,32 @@ export class CourseManagementOverviewStatisticsComponent implements OnInit, OnCh
     loading = true;
     graphType: Graphs = Graphs.ACTIVE_STUDENTS;
 
-    // Chart
-    chartName: string;
-
     // Histogram-related properties
-    barChartOptions: any = {};
-    barChartType: ChartType = 'line';
     amountOfStudents: string;
-    barChartLegend = false;
 
     // Data
-    barChartLabels: Label[] = [];
-    chartData: ChartDataSets[] = [];
-    dataForSpanType: number[] = [];
+    lineChartLabels: string[] = [];
+
+    // ngx-data
+    ngxData: any[] = [];
+    chartColor: Color = {
+        name: 'vivid',
+        selectable: true,
+        group: ScaleType.Ordinal,
+        domain: ['rgba(53,61,71,1)'],
```

Comment:
nitpick: Would be great to have a comment here or a dedicated variable so you would know what color this is